src/main/java/Deadline.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * This represents the subclass Deadline under superclass Task in each element of Task[] list in Duke.java.
+ */
+public class Deadline extends Task {
+
+    protected String by;
+
+    /**
+     * This function initialises the deadline.
+     *
+     * @param description description input by user.
+     * @param by date to complete event by
+     */
```

Comment:
Good job implementing JavaDocs ahead of time! However, all JavaDocs for methods should start with a verb such as `Modifies`, `Gets`, and `Calculates`.